auto_causality/models/__init__.py

```diff
@@ -1,7 +1,43 @@
-from .wrapper import DoWhyWrapper
-from auto_causality.transformed_outcome import TransformedOutcomeFitter
+from typing import List, Union
+
+import pandas as pd
+import numpy as np
+
+
+from auto_causality.scoring import ate
+from auto_causality.models.wrapper import DoWhyWrapper, DoWhyMethods
+from auto_causality.models.transformed_outcome import TransformedOutcomeFitter
 
 
 class TransformedOutcome(DoWhyWrapper):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, inner_class=TransformedOutcomeFitter, **kwargs)
+
+
+class Dummy(DoWhyWrapper):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, inner_class=DummyModel, **kwargs)
+
+
+class DummyModel(DoWhyMethods):
```

Comment:
We should avoid having anything other than imports and a `__all__` in the `__init__.py`s. No one ever thinks to look in here so it can be a source of problems later.